### DISCUSSION TITLE
[IDEA] Infinite scroll capability implementation for product listing

### DISCUSSION THREAD

**Discussion by Dreimus** [PrestaShop/hummingbird#815] [Ideas]
Is it possible to implement a Typescript Component that allow infinite scroll capability in category page ?
We could potentially set a parameter to enable the infinite scroll instead of classic pagination ?

**Comment by eveneme** [comment]
+1

**Comment by yannicka** [comment]
+1 for a parameter to do this.

**Comment by eveneme** [comment]
+1 good idea

**Comment by SharakPL** [comment]
But only with a `Load more` button. Otherwise it will simply prevent access to everything below the list of products until all pages are loaded.

**Comment by Hlavtox** [comment]
Watch out that this needs very good implementation.
Changing the URL to keep track of history and when going back to the page, the user must land on where he was before.
